### PR TITLE
Add Phasespace Motion Capture descriptor

### PR DESCRIPTION
**Pull request by godbyk**

### CODE REVIEW

File: apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "deviceVendor": "PhaseSpace",
+    "deviceName": "Motion Capture",
+    "author": "Kevin M. Godby <kevin@godby.org>",
+    "version": 1,
+    "lastModified": "2016-12-06T22:17:44Z",
+    "interfaces": {
+        "analog": {
```
Comment (rpavlik) [inline apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json]: Unless this actually reports analogs and buttons (which source inspection suggests it does not), remove the analog and button sections - their presence is enough to trigger path creation.

File: apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            "count": 0
+        },
+        "button": {
+            "count": 0
+        },
+        "tracker": {
+            "count": 1
```
Comment (rpavlik) [inline apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json]: Trackers, it is fine to not know the count, so please remove it.

File: apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json (hunk 1 above)
Comment (godbyk) [inline apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json]: I think the `0`s trigger a validation failure, too.. I just haven't set up the validator on this computer yet.

File: apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json (hunk 2 above)
Comment (godbyk) [inline apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json]: @rpavlik You want an empty `tracker` dictionary?

**Comment by rpavlik** [comment]
LGTM

**Pull request merged**